### PR TITLE
CSPL-2409: Splunk Operator for Kubernetes reorders environment variables

### PR DESCRIPTION
**Pull request by vivekr-splunk**
preference will be given to ExtraVars . where anything defined in ExtraVars will be added first in the environment variables. 
any duplicate will be removed

### CODE REVIEW

File: docs/artifacthub-repo.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Some settings like the verified publisher flag or the ignored packages won't
+# be applied until the next time the repository is processed. Please keep in
+# mind that the repository won't be processed if it has not changed since the
+# last time it was processed. Depending on the repository kind, this is checked
+# in a different way. For Helm http based repositories, we consider it has
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+repositoryID: The ID of the Artifact Hub repository where the packages will be published to (optional, but it enables verified publisher)
```
Comment (akondur) [inline docs/artifacthub-repo.yml:12]: Are these changes intended?

**Review by akondur — APPROVED** [review]

File: docs/artifacthub-repo.yml (hunk 1 above)
Comment (vivekr-splunk) [reply docs/artifacthub-repo.yml:12]: yes default template used here

File: docs/index.yaml [hunk 2]
```diff
@@ -86,34 +86,4 @@ entries:
...
-    appVersion: 2.1.0
-    created: "2023-07-25T15:50:32.747409-07:00"
-    description: A Helm chart for the Splunk Operator for Kubernetes
-    digest: 34e5463f8f5442655d05cb616b50391b738a0827b30d8440b4c7fce99a291d9a
-    name: splunk-operator
-    type: application
-    urls:
-    - https://splunk.github.io/splunk-operator/splunk-operator-1.0.0.tgz
-    version: 1.0.0
-generated: "2023-07-25T15:50:32.661115-07:00"
```
Comment (gaurav-splunk) [inline docs/index.yaml:119]: I hope these changes are correct i.e. in line with whatever we have on latest develop/main?

Comment (vivekr-splunk) [reply docs/index.yaml]: yes the only thing done here is to regenerate these files.

**Review by gaurav-splunk — APPROVED** [review]
